src/assets/pages/blogposts/post-48.html

```diff
@@ -0,0 +1,33 @@
+<img src="https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/post0048/post-48-grafico-1.png">
+<h2>Aprendizados</h2>
+<p>Devido ao foco em novos repositórios para recebimento de contribuições, alguns aprendizados aconteceram na prática e devem ser levados em consideração na Hacktoberfest de 2024.</p>
+<h3>Frontend</h3>
+<p>Lucas Guimarães, mantenedor do repositório querido-diario-frontend, destacou que nesse repositório será importante ficar atento não apenas para a quantidade de contribuições, mas também para a qualidade do que está sendo desenvolvido:</p>
+<p class="quote-paragraph"><q>Houve novas adições ao produto vindas de sugestões de novas funcionalidades feitas por colaboradores recém-chegados ao projeto, sem que necessariamente as sugestões propostas estivessem alinhadas com a gestão e fizessem sentido como um todo. No ano que vem vamos precisar utilizar melhor as labels para especificar as características da issue e tentar estabelecer um mecanismo de validar as entradas com os mantenedores, aceitando-as ou não</q> reforça Lucas Guimarães. </p>
+<p>Para lidar com essa questão, Pedro Guimarães reforça que <q>Ano que vamos ter uma melhor gestão prévia do repositório, com preparação e criação de issues e maior dedicação na aplicação da label 'good first issue', que indica as issues mais fáceis</q>, explica.</p>
+<h2>Feedbacks da comunidade</h2>
+<p>No geral, os feedbacks foram bastante positivos para a participação do Querido Diário na Hacktoberfest deste ano, em especial com relação ao engajamento das pessoas nos eventos promovidos e movimentação no servidor. Para Juliana Trevine, a formação de <a href="https://queridodiario.ok.org.br/blog/post/41" target="_blank">grupos de trabalho</a> voltados para áreas específicas do projeto pode ter ajudado. Três GTs iniciaram atividades em agosto deste ano e seguem até o final deste mês. A expectativa é que essa dinâmica continue fazendo parte da comunidade daqui para frente.</p>
+<p>Muita gente nova também ingressou na comunidade durante a Hacktoberfest. Somente entre os dias 3 e 30 de outubro, 78 novas pessoas entraram no <a href="https://go.ok.org.br/discord" target="_blank">servidor da OKBR</a> no Discord!</p>
+<p class="quote-paragraph"><q>"Essa foi a Hacktoberfest com maior engajamento da comunidade que o Querido Diário já teve! A cobertura de municípios quase dobrou em um mês. Nos preparamos para um grande evento, mas os resultados superaram todas as expectativas!"</q><p> celebra Giulio Carvalho, coordenador de inovação cívica na OKBR.</p>
```

Comment:
```suggestion
<p class="quote-paragraph"><q>Essa foi a Hacktoberfest com maior engajamento da comunidade que o Querido Diário já teve! A cobertura de municípios quase dobrou em um mês. Nos preparamos para um grande evento, mas os resultados superaram todas as expectativas!</q> celebra Giulio Carvalho, coordenador de inovação cívica na OKBR.</p>
```
Tinha um `<p>` extra aqui